src/olympia/migrations/938-drop-l10n-from-bio-user-profile.sql

```diff
@@ -0,0 +1,23 @@
+/* FOREIGN KEY is prod's - you will need to change this for your local DB */
+
+/* Add the new column and drop the constraint so we can clear the translations.*/
+ALTER TABLE `users`
+    ADD COLUMN `_bio` longtext NULL,
+    DROP FOREIGN KEY `users_ibfk_1`;
+
+/* Copy across the translated values.  There are some users that have multiple
+localizations so we're going to use the default locale (lang) localization. */
+UPDATE `users`
+    SET `_bio` = (
+        SELECT `localized_string` FROM `translations`
+        WHERE `id`=`bio` AND `locale`=`lang`);
```

Comment:
This isn't necessary but it'd be easier to follow along if we could explicitly use the tables names to avoid ambigueness between `id` and `bio` as well as `locale` and `lang` here.

```sql
UPDATE `users`
    SET `users`.`_bio` = (
        SELECT `localized_string` FROM `translations`
        WHERE `translations`.`id` = `users`.`bio` AND
              `translations`.`locale` = `users`.`lang`);
```